packages/main/test/pages/Switch.html

```diff
@@ -42,6 +42,11 @@ <h3>Default Switch</h3>
 	</div>
 	<ui5-label id="lbl"></ui5-label>
 
+	<h3>Switch with required attribute</h3>
```

Comment:
Could be "Required switch" and if not used for the tests I think we could only have an instance in form as sample.